types/react-native/test/index.tsx

```diff
@@ -922,19 +923,25 @@ const PickerTest = () => (
     </Picker>
 );
 
+const NativeBridgedComponent = requireNativeComponent<{ nativeProp: string }>("NativeBridgedComponent"); // $ExpectType HostComponent<{ nativeProp: string }>
+
 class BridgedComponentTest extends React.Component {
     static propTypes = {
         jsProp: PropTypes.string.isRequired,
         ...ViewPropTypes,
     };
 
+    nativeComponentRef: typeof NativeBridgedComponent
```

Comment:
Can you wire this up a bit more? Because in practice this field has to be nullable requiring a null check for the method can be called.

Attach a ref callback to the function below that sets this class field. TypeScript should enforce that this field has to be nullable and that you have to do the null check.

Reply:
Done, good call 👍